src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -1291,38 +1248,40 @@ public static string GetModulePath(string currentProcessModulePath, string hklmM
                 {
                     currentProcessModulePath += hklmMachineModulePath; // += EVT.Machine
                 }
+
+#if !UNIX
+                // Add Windows Modules path
+                currentProcessModulePath += Path.PathSeparator;
```

Comment:
Is it possible to tweak the way this string is built so it's not reallocated twice? Maybe with `$"{}"` syntax?

Reply:
Will update